exercises/resistor-color/src/example.cr

```diff
@@ -0,0 +1,13 @@
+module ResistorColor
+  extend self
+
+  @@colors = ["black", "brown", "red", "orange", "yellow", "green", "blue", "violet", "grey", "white"]
```

Comment:
I would think that this should be a constant but that might just be a carryover from my Ruby knowledge (might not matter in Crystal; and there may not be any perf improvements)... @bmulvihill , any thoughts?

Reply:
From looking at code examples online, I have seen both used in different cases. I kind of feel like a constant could be a good update for this. Let me know what you want and I'll make the update! Thanks!

Reply:
@mhelmetag I think you are right, this feels better as a constant.